src/SDKs/CognitiveServices/dataPlane/Vision/FormRecognizer/FormRecognizer.Tests/Microsoft.Azure.CognitiveServices.FormRecognizer.Tests.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
```

Comment:
For consistency please rename this file to `CognitiveServices.FormRecognizer.Tests.csproj`

Reply:
done.